.github/workflows/e2etest.yaml

```diff
@@ -108,31 +113,31 @@ jobs:
               "environment": [
                 {
                   "name": "platformEnv",
-                  "value": "${{ inputs.platform_env }}"
+                  "value": "${{ github.event.inputs.platform_env || inputs.platform_env }}"
                 },
                 {
                   "name": "browserName",
-                  "value": "${{ inputs.browser_name }}"
+                  "value": "${{ github.event.inputs.browser_name || inputs.browser_name }}"
                 },
                 {
                   "name": "browserVersion",
-                  "value": "${{ inputs.browser_version }}"
+                  "value": "${{ github.event.inputs.browser_version || inputs.browser_version }}"
                 },
                 {
                   "name": "exclude_tags",
-                  "value": "${{ inputs.exclude_tags }}"
+                  "value": "${{ github.event.inputs.exclude_tags || inputs.exclude_tags }}"
                 },
                 {
                   "name": "cucumberTags",
-                  "value": "${{ inputs.cucumber_tags }}"
+                  "value": "${{ github.event.inputs.cucumber_tags || inputs.cucumber_tags }}"
                 },
                 {
                   "name": "buildId",
                   "value": "${{ env.JOB_NAME }}"
                 },
                 {
                   "name": "mavenOptions",
-                  "value": "${{ inputs.maven_options }}"
+                  "value": "${{ github.event.inputs.maven_options || inputs.maven_options }}"
```

Comment:
Isn't `github.event.inputs` the same as `inputs` in this context?

Reply:
Thought we needed it if we are to invoke directly from the mono-repo, but if we are fine with just inputs can revert it back

Reply:
All good from the mono-repo, we can use `inputs` directly for both events.